### ISSUE TITLE
Is the project still maintained?

### ISSUE THREAD

**Issue by guylando** [HubSpot/odometer#154]
I see Odometer was last updated 1 year ago and there are 65 open issues versus 55 closed issues, etc.
All brings to the conclusion that the project was abandoned and it is not maintained anymore.

Can someone confirm this? Can anyone suggest a popular good alternative library offering nice animated timers and which is actively developed and maintained?

**Comment by coderitual** [comment]
I don't now tbh. I wanted to use this lib some time ago but unfortunately I wasn't able due to lack of features I needed, so I created my own library - [bounty](https://github.com/coderitual/bounty). Both libs are sligtly different in areas they want to cover so I don't think there is one-to-one replacement for odometer. I focused more on particular effect.

@adamschwartz, @zackbloom  don't treat this as a competition, your lib is a way more comprehensive :)

**Comment by javascript-pro** [comment]
TBH, I've used it successfully in a React app with great results. I think it is stable enough to do any kind of odometer rewuirement, you'd just have to slot it in correctly

**Comment by GarrettJMU** [comment]
@listingslab Did you use the npm or implement this?

**Comment by javascript-pro** [comment]
no, it was a bit less elegant than that. I included the minimised script in the HTML and used it from within react using a globally accessible variable arroach
